Source/WebKit/NetworkProcess/NetworkDataTask.cpp

```diff
@@ -180,7 +180,7 @@ NetworkSession* NetworkDataTask::networkSession()
 void NetworkDataTask::restrictRequestReferrerToOriginIfNeeded(WebCore::ResourceRequest& request)
 {
 #if ENABLE(TRACKING_PREVENTION)
-    if ((m_session->sessionID().isEphemeral() || m_session->isResourceLoadStatisticsEnabled()) && m_session->shouldDowngradeReferrer() && request.isThirdParty())
+    if ((m_session->sessionID().isEphemeral() || m_session->istrackingPreventionEnabled()) && m_session->shouldDowngradeReferrer() && request.isThirdParty())
```

Comment:
```suggestion
    if ((m_session->sessionID().isEphemeral() || m_session->isTrackingPreventionEnabled()) && m_session->shouldDowngradeReferrer() && request.isThirdParty())
```

camelCase is not respected here.

Reply:
Fixed all of these.